chart/templates/timescaledb-toolkit.yaml

```
@@ -0,0 +1,18 @@
+{{- $tsdb := index .Values "timescaledb-single" -}}
+{{ if $tsdb.enabled -}}
+{{- /*
+FIXME(paulfantom): Remove this ConfigMap when timescaleDB images start to be published with toolkit pre-enabled
+*/}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: custom-init-scripts
+  namespace: {{ template "tobs.namespace" . }}
+  labels:
+   app: {{ template "tobs.fullname" . }}
+   chart: {{ template "tobs.chart" . }}
+   release: {{ .Release.Name }}
+data:
+  toolkit.sql: |
+    create extension if not exists timescaledb_toolkit
```

small nit, maybe not even required.

```suggestion
    create extension if not exists timescaledb_toolkit;
```